### PR TITLE
Backport the frame ancestors self switch to this plugin

### PR DESCRIPTION
**Pull request by zero-24**
Hi,

this is a backpot for the PR i made for the core version here: https://github.com/joomla/joomla-cms/pull/28066

@brianteeman @Sandra97 @YGomiero @jeckodevelopment @pe7er please send me your translations / suggestions for the new language string as a comment or [suggestion](https://haacked.com/archive/2019/06/03/suggested-changes/) to this PR so I can add it to this PR and the final release.

```
PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY_FRAME_ANCESTORS_SELF_ENABLED_DESC="Enable the CSP clickjacking protection frame-ancestors and only allow the origin 'self'. Please use the form below to allow origins other than 'self'."
```

Feel free to ping me on any questions.

### CODE REVIEW

File: language/it-IT/it-IT.plg_system_httpheader.ini [hunk 1]
```diff
@@ -11,6 +11,8 @@ PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER_KEY="Header HTTP"
 PLG_SYSTEM_HTTPHEADER_ADDITIONAL_HEADER_VALUE="Valore Header HTTP"
 PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY="<a href='https://scotthelme.co.uk/content-security-policy-an-introduction' target='_blank' rel='noopener noreferrer'>Content Security Policy (CSP)</a>"
 PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY_DESC="Con quest'opzione tutti i valori per la 'Content-Security-Policy' possono essere impostati individualmente."
+PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY_FRAME_ANCESTORS_SELF_ENABLED="frame-ancestors 'self'"
+PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY_FRAME_ANCESTORS_SELF_ENABLED_DESC="Enable the CSP clickjacking protection frame-ancestors and only allow the origin 'self'. Please use the form below to allow origins other than 'self'."
```
Comment (jeckodevelopment) [inline language/it-IT/it-IT.plg_system_httpheader.ini]: Abilita la protezione CSP da clickjacking frame-ancestors e consenti solo l'origine 'self'. Utilizza il modulo sottostante per abilitare altre origini diverse da 'self'.

**Review by jeckodevelopment — CHANGES_REQUESTED** [review]

**Comment by jeckodevelopment** [comment]
Hope that this way was correct :)

**Comment by Sandra97** [comment]
Activer la protection CSP frame-ancestors contre le détournement de clic et n'autoriser que l'origine "self". Veuillez utiliser le formulaire ci-dessous pour autoriser les origines autres que "self".

**Comment by pe7er** [comment]
PLG_SYSTEM_HTTPHEADER_CONTENTSECURITYPOLICY_FRAME_ANCESTORS_SELF_ENABLED_DESC="Schakel de CSP clickjacking bescherming voor frame-ancestors in en sta alleen 'zelf' als bron toe. Gebruik het onderstaande formulier om andere bronnen dan 'zelf' toe te staan."

**Comment by zero-24** [comment]
Many thanks I have just committed the updated language strings and can do a release soon 👍 

ps @pe7er i have used `'self'` over your `'zelf'` as `'self'` is the actual value you set to the header.

**Pull request merged**